lib/bitstyles_phoenix/component/form.ex

```diff
@@ -23,6 +23,7 @@ defmodule BitstylesPhoenix.Component.Form do
     color: :color_input,
     checkbox: :checkbox,
     date: :date_input,
+    datetime: :datetime_local_input,
```

Comment:
```suggestion
    datetime_local: :datetime_local_input,
```
Let's make a 1 to 1 mapping then. Maybe we come up with a nice little js datetime input later (since the datetime_local one still has it's problems)